x-pack/plugins/apm/common/runtime_types/date_as_timestamp_rt/index.test.ts

```diff
@@ -5,23 +5,23 @@
  * 2.0.
  */
 
-import { dateAsStringRt } from './index';
+import { dateAsTimestampRt } from './index';
 import { isLeft, isRight } from 'fp-ts/lib/Either';
 
-describe('dateAsStringRt', () => {
+describe('dateAsTimestampRt', () => {
   it('validates whether a string is a valid date', () => {
-    expect(isLeft(dateAsStringRt.decode(1566299881499))).toBe(true);
+    expect(isLeft(dateAsTimestampRt.decode(1566299881499))).toBe(true);
 
-    expect(isRight(dateAsStringRt.decode('2019-08-20T11:18:31.407Z'))).toBe(
+    expect(isRight(dateAsTimestampRt.decode('2019-08-20T11:18:31.407Z'))).toBe(
       true
     );
   });
 
   it('returns the string it was given', () => {
-    const either = dateAsStringRt.decode('2019-08-20T11:18:31.407Z');
+    const either = dateAsTimestampRt.decode('2019-08-20T11:18:31.407Z');
 
     if (isRight(either)) {
-      expect(either.right).toBe('2019-08-20T11:18:31.407Z');
+      expect(either.right).toBe(1566299911407);
     } else {
       fail();
     }
```

Comment:
Can you also add a test for encoding (since you implemented the encoder function).

Reply:
`.isLeft()` and `.isRight()` can be replaced by `.is()` (I didn't know about that until I saw it in some code from the infra team)

Reply:
@dgieselaar can you point me where you saw it? I didn't find any `is()` function declared on io-ts

Reply:
it's `myCustomRt.is()`